src/libs/style.js

```diff
@@ -88,23 +88,23 @@ export const cardList = {
   }
 }
 
-export const navList = {
+export const findWorkflowNavList = {
   heading: {
     color: colors.dark(), backgroundColor: colors.light(0.4), fontSize: 16, padding: '1rem 1.5rem',
     display: 'flex', justifyContent: 'space-between', alignItems: 'center', margin: 0,
     fontWeight: 600, textTransform: 'uppercase', borderBottom: `0.5px solid ${colors.dark(0.2)}`
   },
   itemContainer: selected => ({
     display: 'flex', alignItems: 'center', flex: 'none', width: '100%', height: 50,
-    padding: '0 1.5rem', backgroundColor: 'white',
```

Comment:
Will this have unintended consequences in our other workflow nav lists? (presumably... just the main landing page?)

Reply:
It turned out that this `navList` styling wasn't used in CBAS UI anywhere (yet). Hence I just removed these lines. My assumption is in future when we do need it we can have general `navList` style again with `findWorkflowNavList` only editing what it doesn't need.